elephant/signal_processing.py

```diff
@@ -134,6 +135,111 @@ def zscore(signal, inplace=True):
         return result[0]
     else:
         return result
+    
+    
+def cross_correlation_function(signal, ch_pairs, dt=1., env=False, nlags=None):
+
+    """
+    Computes unbiased estimator of the cross-correlation function.
+    
+    Calculates the unbiased estimator of the cross-correlation function 
+    R(tau) = 1/(N-|k|) R'(tau), where R'(tau) = E[x(t)*y(t+tau)] for signal1 
```

Comment:
Can you add a reference for this normalization? Also please add a description of the variables you are using.

Reply:
Will add: Hall & River (2009) Spectral Analysis of Signals, Spectral Element Method in Structural Dynamics, Eq. 2.2.3